### DISCUSSION TITLE
登陆失败,报错237,使用核心:Mirai-2.15-M1

### DISCUSSION THREAD

**Discussion by NekoEpisode** [DreamVoid/MiraiMC#391] [问答 / Q&A]
Error(bot=Bot(2810140935), code=237, title=安全提醒, message=当前网络不稳定，登录失败。推荐使用常用设备或通过手机号登录。, errorInfo=)
是MiraiMC的问题吗?

**Comment by DreamVoid** [comment]
是你的问题
